.gitignore

```diff
@@ -37,3 +37,5 @@ bin
 
 .idea/
 *.iml
+
+.vscode
```

Comment:
I guess this makes sense, but:
1. it shouldn't be in this PR
2. it is recommended to use your own global gitignore for those things

My guess is that you didn't realize it sneaked in here.  Could you remove it?

Reply:
Done.  Thanks for suggesting this.  I sort of always knew this was an option, but was too lazy to set it up.

Reply:
If it helps: `git config --global core.excludesfile ~/.gitignore`